docs/general/ddl/data-types.rst

```diff
@@ -40,7 +40,7 @@ or collections.
 * `bigint <numeric types_>`_
 * `real <numeric types_>`_
 * `double precision <numeric types_>`_
-* `text <data-type-text>`_
+* `text <data-type-text_>`_
```

Comment:
Could we remove the trailing whitespace in the marker declaration instead?

Reply:
Not sure I am following here. Which trailing whitespace are you referring to?

Reply:
Ah sorry, due to the Github diff being underlined already I thought there was a whitespace at the end instead of an underscore. Nevermind :)